src/mixins/audio_token_controls.js

```diff
@@ -15,10 +15,26 @@
   var playIconOffsetX = 50;
   var playIconOffsetY = 50;
 
+
   if (fabric.Audio_token) {
 
     var audioTokenControls = fabric.Audio_token.prototype.controls = { };
     // create custom audio_token control for delete icon
+
+    function drawFocusOnPlayButton(ctx, left, top, size) {
+      //this is fabric audio token object here
+      ctx.fillStyle = this.focusStrokeOuter;
```

Comment:
we should save the context and the restore it at the end so that we don't bleed the fillStyle here.
for example.
ctx.save();
..
..
..
ctx.restore();

Reply:
fixed, thanks!